packages/manager/modules/octavia-load-balancer/src/create/translations/Messages_fr_FR.json

```
@@ -10,14 +10,15 @@
   "octavia_load_balancer_create_size_flavour_description_l": "Lorsque le trafic de vos applications atteint sa plus haute densité, vous pouvez avoir besoin des meilleures performances du Load Balancer pour y faire face.",
   "octavia_load_balancer_create_size_flavour_price_interval": " / heure",
   "octavia_load_balancer_create_region_title": "Sélectionnez une région",
-  "octavia_load_balancer_create_region_intro": "Veuillez sélectionner la région Public Cloud où votre Load Balancer (no translation) sera hébergé. Parmi les regions public cloud où un réseau privé est défini, sélectionnez celle où le LB sera hébergé. Vous ne trouvez pas la région que vous désirez dans la liste ? Pour en savoir plus sur la disponibilité des fonctions par région, ",
+  "octavia_load_balancer_create_region_intro": "Veuillez sélectionner la région Public Cloud où votre Load Balancer sera hébergé. Parmi les regions public cloud où un réseau privé est défini, sélectionnez celle où le LB sera hébergé. Vous ne trouvez pas la région que vous désirez dans la liste ? Pour en savoir plus sur la disponibilité des fonctions par région, ",
```

```suggestion
  "octavia_load_balancer_create_region_intro": "Veuillez sélectionner la région Public Cloud où votre Load Balancer (LB) sera hébergé. Parmi les regions public cloud où un réseau privé est défini, sélectionnez celle où le LB sera hébergé. Vous ne trouvez pas la région que vous désirez dans la liste ? Pour en savoir plus sur la disponibilité des fonctions par région, ",
```
or
```suggestion
  "octavia_load_balancer_create_region_intro": "Veuillez sélectionner la région Public Cloud où votre Load Balancer sera hébergé. Parmi les regions public cloud où un réseau privé est défini, sélectionnez celle où le Load Balancer sera hébergé. Vous ne trouvez pas la région que vous désirez dans la liste ? Pour en savoir plus sur la disponibilité des fonctions par région, ",
```
If we want to use abbreviations, I think we should quote them above.
I think the first and beginning of the second sentences are duplicates.

---

Is it our team responsability to change the content ?